packages/shared/src/components/filters/TagsFilter.spec.tsx

```diff
@@ -141,24 +148,23 @@ it('should show login when not logged in', async () => {
   expect(showLogin).toBeCalledTimes(1);
 });
 
-it('should follow a tag on click', async () => {
-  let mutationCalled = false;
+it('should follow a tag on click and remove filter alert if enabled', async () => {
+  const disableAlertFilterMock = jest.fn();
+  const addFilterMutation = jest.fn();
 
-  const { baseElement } = renderComponent();
+  const { baseElement } = renderComponent(
+    [createAllTagCategoriesMock()],
+    defaultUser,
+    { alerts: { filter: true }, disableFilterAlert: disableAlertFilterMock },
+  );
 
-  await waitFor(async () => {
-    const data = await client.getQueryData(
-      getFeedSettingsQueryKey(defaultUser),
-    );
-    expect(data).toBeTruthy();
-  });
   mockGraphQL({
     request: {
       query: ADD_FILTERS_TO_FEED_MUTATION,
       variables: { filters: { includeTags: ['webdev'] } },
     },
     result: () => {
-      mutationCalled = true;
+      addFilterMutation();
```

Comment:
Again why did you change this mechanism?

Reply:
I just kept it to be consistent with each other but we can revert it if necessary.